setup.py

```diff
@@ -14,7 +14,7 @@
 REQUIREMENTS = [
     'httplib2 >= 0.9.1',
     'googleapis-common-protos',
-    'oauth2client >= 1.4.6',
+    'oauth2client >= 2.0.0',
```

Comment:
ISTM that bumping a dependency to a new major version (non-backward-compatible) likely means we need to bump our version as well (e.g, `0.10` rather than `0.9.1`).